### DISCUSSION TITLE
Slider's labels turns to numbers wen width is small enough.

### DISCUSSION THREAD

**Discussion by CmdrMahesh** [fomantic/Fomantic-UI#1634] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello Guys,

I have countered an issue with the slider module.

![want](https://user-images.githubusercontent.com/46396430/89652574-4b41ca00-d8e3-11ea-8639-581e1ea8430d.png)
As we can see here the labels are visible and look cool.

![having](https://user-images.githubusercontent.com/46396430/89652652-6c0a1f80-d8e3-11ea-8fe9-05a09a23f323.png)
And in the above image, the same slider is showing numbers if we shrink it in terms of width.

So is there any way to show the labels even if we shrink it down and not the numbers which are meaningless.

Thanks in advance, your support is awesome as always.

**Comment by lubber-de** [comment]
I cannot reproduce this.
See jsfiddle here https://jsfiddle.net/lubber/3e8tmxdj/6/

![sliderKeep](https://user-images.githubusercontent.com/18379884/89654116-75d25980-d8c8-11ea-939b-bc5aa55591d3.gif)


I believe you are working with float numbers and the interpretLabel function cannot handle them when shrinking size resulting into values which arent fetched.

**Comment by CmdrMahesh** [comment]
@lubber-de ,

Thank you for the fast reply.
Please visit my [demo server](https://demo.woochoiceplugin.com/hify-store/product-category/eo_diamond_shape_cat/?EO_WBC=1&BEGIN=eo_diamond_shape_cat&STEP=1&FIRST=&SECOND=) where you can find the color labeled slider.

![captured](https://user-images.githubusercontent.com/46396430/89656128-c2c62800-d8e8-11ea-98a3-dcf7eaa0a988.gif)
The above image shows the issue I am facing.

Thank you.

**Comment by lubber-de** [comment, marked as answer]
Please provide a jsfiddle, we really dont have time to debug your website and find the relevant code....

however, i found your code for `interpretLabel`

```
...
_label_value = _label_value.map(function(_label_value_ele){
--
  | return _label_value_ele[0];
  | });
...
```
The map function does not seem to find the appropriate labels for given float values (which occur when you change the width).
The slider is just displaying the values you are returning from your calculation in `interpretLabel`, so debug this part of the code and change the return value accordingly for all cases.

**Comment by CmdrMahesh** [comment]
@lubber-de ,

Thank you for the hint. That was a great help.

**Comment by lubber-de** [comment]
Closing, because it's not a FUI related core issue
